lib/gui/os/dialog/services/dialog.js

```diff
@@ -113,20 +113,28 @@ module.exports = function($q, SupportedFormatsModel) {
    * });
    */
   this.showWarning = (options) => {
+    const BUTTONS = [
+      options.confirmationLabel,
+      options.rejectionLabel
+    ];
+
+    const BUTTON_CONFIRMATION_INDEX = _.indexOf(BUTTONS, options.confirmationLabel);
+    const BUTTON_REJECTION_INDEX = _.indexOf(BUTTONS, options.rejectionLabel);
```

Comment:
Yeah, I agree. The Electron dialog module makes heavy use of number indexes, which is terrible to deal with: https://electron.atom.io/docs/api/dialog/#dialogshowmessageboxbrowserwindow-options-callback

I guess we can remove the need for `BUTTON_CONFIRMATION_INDEX ` using your suggestion from https://github.com/resin-io/etcher/pull/1149#discussion_r104024047, and only keep the rejection one, which is at least nicer. What do you think?